### PR TITLE
Add blog post: Prescription, billing, and access improvements in CARE

### PR DESCRIPTION
**Pull request by aravindm4**
Adds a new blog post covering the July 2026 CARE release, including:

- Nutrition and medication administration
- Prescriptions (infinite scrolling, date-range dosing)
- Appointments and billing improvements
- Clinical data alignment with FHIR
- Files and uploads (camera capture, plugin extensibility)
- Access and security (phone-based password reset, session purge)
- Usability improvements (patient age component, update prompts)
- Under the hood (Vite plugin registration, Node.js 24)
- Operations and ecosystem (ISO certification, DPG renewal)

### CODE REVIEW

File: content/blog/prescription-billing-access-improvements-care.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+## Usability
+
+Patient age is now shown consistently across encounters, appointments, queues, billing, and patient information through a shared component, with an optional tooltip giving a years, months, and days breakdown. The automatic app update no longer interrupts work on pages other than the home route, showing an update prompt instead. Date and time input styling was unified, and two sidebar issues were fixed so selectors work when the sidebar is collapsed and clicking the current page no longer triggers a needless reload.
+
+## Under the hood
+
+Plugin UI components now register automatically through a Vite plugin rather than manual wiring. The frontend moved to Node.js 24 with a round of dependency upgrades. The month also added test coverage across billing and medication timing logic, hardened the shared request layer against pagination and response parsing edge cases, and introduced an automated pull request review workflow, alongside the usual continuous integration and tooling changes.
```
Comment (billjohnc) [inline content/blog/prescription-billing-access-improvements-care.md]: "The month also added" could be better if we could rephrase this.
